### DISCUSSION TITLE
Andiamo on line?

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#36] [General]
Oggi è uscita la puntata 53, ultima di questa stagione. Ho aggiornato il nostro database con la puntata (titolo, numero, cover, eccetera) ma non avendola ancora ascoltata non ho aggiunto i giochi.

Per @MarcoJul: ho fatto qualche modifica al frontend per quanto riguarda la ricerca e la visualizzazione del numero di episodi perché la pagina era troppo pesante. Prima di procedere per favore fai merge col mio `main` e fai girare `yarn install` perché ho aggiornato tutti i pacchetti del frontend.

@agofa prova un po' [staging](https://staging.euscitojoypad.it/) (dal telefono) e facci sapere se la ricerca è un po' migliorata.

Ma più in generale a me sembra che il nuovo frontend sia pronto da tempo. Che dite, andiamo on line?

**Comment by agofa** [comment]
Su online dico di sì

Nel pomeriggio farò test e sbobinerò (sic) l'episodio

Dai che si va

**Comment by agofa** [comment]
Fatta qualche prova
Stray trova ancora un sacco di epicosi: 17
Per esempio trova il 44 ma non capisco perché. Forse perché c'è "**stay** human""

RIcerca di Elden ring va molto meglio

Su mobile la tastiera nasconde la barra della ricerca mentre si sta scrivendo **PARE SOLO SU FIREFOX PERO'**
![immagine](https://user-images.githubusercontent.com/19308040/181917612-e3cbc972-1e3f-44c2-afd2-282a4ae77281.png)

**Reply by pdonadeo** [reply]
"Stray" è davvero una parola troppo breve e "simile" a molte altre. Come dici probabilmente trova anche "stay".

Però i risultati dovrebbero essere in ordine di importanza. L'episodio 44, quello che non c'entra niente, è tra i primi risultati o tra gli ultimi? Se è tra i primi c'è poco da fare se non cambiare completamente tecnologia.

**Reply by agofa** [reply]
Direi che potrebbe andar bene.
I primi due della lista sono il 53 e il 13-1, in entrambi c'è il gioco
Poi il 34, ci sono giochi che iniziano con "stra" (Death Stranding, Stranger of paradise)
Quindi il 50 (di nuovo death stranding)
Il 38 (death stranding e life is strange)
Il 44 è penultimo, ultimo il 33, entrambi con Stay Human


Insomma, per me si può aprire al mondo

**Reply by pdonadeo** [reply]
Domani faccio un po' di esperimenti per capire se si può tagliare via dal risultato i match più scarsi.

**Reply by agofa** [reply]
Nel frattempo sto facendo test a tempo perso. Direi che la ricerca funziona bene
Un paio di esempi

"Human" trova due episodi, con Humankind e Too Human
Ho controllato nel retrobottega e in magazzino non c'è altro che ci assomigli :-)

"ghost" trova un sacco di episodi, ma in assortimento abbiamo almeno 6 prodotti con ghost nel nome
Ho guardato a campione e tutti quelli che ho controllato hanno un gioco con "ghost" nel nome
